### PR TITLE
Completed prep-javascript-variables.

### PR DESCRIPTION
**Pull request by stefancarrera**
java1

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    console.log(fullName);
+    var shortName = 'Stef Carr';
+    console.log(shortName);
+    var numberOfPets = '1';
+    console.log(numberOfPets);
+    var likesVegetables = true;
+    console.log(likesVegetables);
+    var nothing = null;
+    console.log(null);
+    var never =
```
Comment (thebearingedge) [inline prep-javascript-variables/index.html]: For this variable, you can leave out the `=` entirely.

```js
var never;
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great! Just one small change.

**Comment by stefancarrera** [comment]
java1-5

**Review by thebearingedge — APPROVED** [review]
Nice! 👍

**Pull request merged**